shared/overview/2.x.md

```diff
@@ -12,40 +12,41 @@ The first version of {{ $names.os.lower }} was developed as part of the {{ $name
+### Development vs. Production mode
 
-Each version of {{ $names.os.lower }} is available in development and production variants, both built from the same source, but with slightly differing feature sets.
+{{ $names.os.lower }} can be downloaded in production or development mode. This can be later changed via [developmentMode][config-json-developmentmode]
 
-|   Version Name   | Variant Type |                                                                                                            Description                                                                                                             |
-|:----------------:|:------------:|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
-| 2.46.1+rev3-prod |  production  |                                                          This is the production version of {{ $names.os.lower }} and should be used for any production fleet deployments.                                                          |
-| 2.46.1+rev3-dev  | development  | This is the development version of {{ $names.os.lower }} and should be used when you are developing a new application and want to use the fast [local mode][local-mode] workflow. This variant should never be used in production. |
+The development mode is recommended while getting started with {{ $names.os.lower }} and building an application using the fast [local mode][local-mode] workflow. The development mode enables a number of useful features while developing, namely:
 
-### Development vs. Production images
-
-__Warning:__ Development images have an exposed Docker socket and enable passwordless root SSH access and should never be used in production.
-
-The development images are recommended while getting started with {{ $names.os.lower }} and building an application. The development images enable a number of useful features while developing, namely:
-
-* Passwordless [SSH access][ssh-host] into {{ $names.os.lower }} on port `22222` as the root user.
+* Passwordless [SSH access][ssh-host] into {{ $names.os.lower }} on port `22222` as the root user, unless custom [ssh keys][config-json-ssh] are provided in which case key-based authentication is used.
 * Docker socket exposed on port `2375`, which allows `{{ $names.company.lower }} push` / `build` / `deploy`, that enables remote Docker builds on the target device (see [Deploy to your Fleet][deploy-to-fleet]).
 * Getty console attached to tty1 and serial.
 * Capable of entering [local mode][local-mode] for rapid development of application containers locally.
 
-__Note:__ Raspberry Pi devices don’t have Getty attached to serial.
+__Note:__ Raspberry Pi devices don’t have Getty attached to serial by default, but they can be configured to enable serial in the {{ $names.cloud.lower }} dashboard.
```

Comment:
```suggestion
__Note:__ Raspberry Pi devices don’t have Getty attached to serial by default, but they can be configured to enable serial in the {{ $names.cloud.lower }} Dashboard via [configuration variables][supervisor-configuration-list].
```
Add the link to `/reference/supervisor/configuration-list`

Reply:
Done